chemprop/nn/predictors.py

```diff
@@ -115,30 +123,9 @@ def __init__(
         dropout: float = 0,
         activation: str = "relu",
         criterion: LossFunction | None = None,
-        loc: float | Tensor = 0.0,
-        scale: float | Tensor = 1.0,
+        output_transform: OutputTransform = None,
     ):
-        super().__init__(n_tasks, input_dim, hidden_dim, n_layers, dropout, activation, criterion)
-
-        if isinstance(loc, float):
-            loc = torch.ones(1, self.n_tasks) * loc
-        else:
-            loc = torch.tensor(loc).view(1, -1)
-        self.register_buffer("loc", loc)
-
-        if isinstance(scale, float):
-            scale = torch.ones(1, self.n_tasks) * scale
-        else:
-            scale = torch.tensor(scale).view(1, -1)
-        self.register_buffer("scale", scale)
-
-    def forward(self, Z: Tensor) -> Tensor:
-        Y = super().forward(Z)
-
-        return self.scale * Y + self.loc
-
-    def train_step(self, Z: Tensor) -> Tensor:
-        return super().forward(Z)
+        super().__init__(n_tasks, input_dim, hidden_dim, n_layers, dropout, activation, criterion, output_transform)
```

Comment:
can remove this line if you're just passing everything through

Reply:
I'm a bit confused by what you mean. Do you mean like this?

```
@PredictorRegistry.register("regression")
class RegressionFFN(_FFNPredictorBase):
    n_targets = 1
    _default_criterion = MSELoss()
    _default_metric = MSEMetric()

    def __init__(self, **kwargs,):
        super().__init__(**kwargs)
```

Reply:
yeah you can just delete the method in the child class entirely